js/client-battle-tooltips.js

```diff
@@ -415,7 +415,7 @@ var BattleTooltips = (function () {
 			else if (pokemon.maxhp == 48) exacthp = ' <small>(' + pokemon.hp + '/' + pokemon.maxhp + ' pixels)</small>';
 			text += '<p>HP: ' + pokemon.hpDisplay() + exacthp + (pokemon.status ? ' <span class="status ' + pokemon.status + '">' + pokemon.status.toUpperCase() + '</span>' : '');
 			if (pokemon.statusData) {
-				if (pokemon.status === 'tox') {
+				if (pokemon.status === 'tox' && !(pokemon.ability in {poisonheal:1, magicguard:1})) {
```

Comment:
Is `pokemon.ability` an ability ID? I thought it was an ability name...

Reply:
oh shit u right